doc/BUILDSERVER.md

```diff
@@ -162,3 +165,44 @@ If you have issues that are related to the build system you can open a normal
 issue and tag it with `build` and `question`.
 If you feel like your inquiry does not warrent a issue on its own, please use
 [our buildserver issue](https://issues.libelektra.org/160).
+
+## Jenkins
+This section describes how to replicate the current jenkins configuration.
+
+### Jenkins libelektra configuration
+The `libelektra` build job is a multibranch pipeline job.
```

Comment:
Please add a link to the "multibranch pipeline job" plugin. What are the main plugins in use?